builtin/providers/docker/resource_docker_container.go

```diff
@@ -2,10 +2,14 @@ package docker
 
 import (
 	"bytes"
+	"crypto/sha1"
+	"encoding/hex"
 	"fmt"
-
+	"io"
+	"os"
 	"regexp"
 
+	"github.com/fsouza/go-dockerclient/external/github.com/docker/docker/pkg/archive"
```

Comment:
Not sure whether transitive dependencies are the way to go here. Any suggestions?